README.md

```diff
@@ -135,6 +135,25 @@ my-teradata-db-profile:
       port: <port>
 ```
 
+### Retries
+
+Allows an adapter to automatically try again when the attempt to open a new connection on the database has a transient, infrequent error. This option can be set using the `retries` configuration.
+Default value is 0.
```

Comment:
@tallamohan, I feel, it is better to mention on number times retires based on retries value.